leakcanary-android/src/main/java/com/squareup/leakcanary/internal/LeakCanaryInternals.java

```diff
@@ -50,6 +52,8 @@
 
   private static final Executor fileIoExecutor = newSingleThreadExecutor("File-IO");
 
+  private static final String NOTIFICATION_CHANNEL_ID = "leakcanary";
```

Comment:
The collision would still be theoretically possible, albeit less likely with a longer name. However, I'm happy to change this constant if it improves the overall quality of the project.